server/api/cvrs.py

```diff
@@ -68,7 +68,9 @@ def process():
 
         # Parse out all the initial metadata
         _election_name = next(cvrs)[0]
-        contest_row = next(cvrs)
+        contest_row = [
+            contest.replace("\r", "").replace("\n", "  ") for contest in next(cvrs)
```

Comment:
if you wanted to do it in one function call, you could do `re.sub(r"\r*\n+", " ", contest)`

Reply:
actually, a better option might be `" ".join(contest.splitlines())` because splitlines is very smart about handling all types of newlines

Reply:
oooh that's a great suggestion